### ISSUE TITLE
[Question] .net core 3.1 issue during certificate revocation checks 

### ISSUE THREAD

**Issue by dv00d00** [natemcmaster/LetsEncrypt#80]
I have experienced quite a bug several weeks ago, after investigation it looks like the issue might be in the .net core itself. It does not directly involve this project, posting it here just in case.

Full details here: https://github.com/dotnet/aspnetcore/issues/21183. 

The code which fixes it:
```c#
            var chain = new X509Chain
            {
                ChainPolicy =
                {
                    VerificationFlags = X509VerificationFlags.AllFlags,
                    RevocationFlag = X509RevocationFlag.ExcludeRoot,
                    RevocationMode = X509RevocationMode.NoCheck
                }
            };
            
            if (chain.Build(certificate))
            {
                return certificate;
            }
            else
            {
                chain.Dispose();
                return null;
            }
```

Let me know if you are interested in pr / help in implementing this workaround

**Comment by natemcmaster** [comment]
Hey @dv00d00  - I looked at your https://github.com/dotnet/aspnetcore/issues/21183, and it looks like you're not using this project. Did you mean to open this on the fluffyspoon project?

> Let me know if you are interested in pr / help in implementing this workaround

Yes, always open for PRs!

**Comment by natemcmaster** [comment]
Nevermind, I see you're also working with the fluffyspoon project. Thanks for reaching out. David also reached out to me. Seems like this problem should be easy for fix now that we know what is going on with the x.509 checks

**Comment by natemcmaster** [comment]
Thanks for the fix @dv00d00! This will be released soon in 0.5.0
